build.gradle

```diff
@@ -107,7 +104,6 @@ rat {
     "sdks/python/NOTICE",
```

Comment:
I cleaned up the duplication in https://github.com/rmannibucau/incubator-beam/pull/1, please consider merging into your branch.